tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -37,6 +37,7 @@ describe('package/', () => {
       '!**/*.test.*',
       '!**/__snapshots__/',
       '!**/__snapshots__/**',
+      '!src/govuk-esm/_govuk-prototype-kit-init.js',
```

Comment:
This doesn't seem to mach any paths introduced in this PR – left over from previous versions?